cockroachcloud/learn-about-pricing.md

```diff
@@ -23,6 +23,8 @@ RU and storage consumption is prorated at the following prices:
 Your cluster's [resource limits](../{{site.versions["stable"]}}/architecture/glossary.html#resource-limits) are the maximum amount of storage and RUs you can use in a month. If you reach your storage limit, your cluster will be throttled and you will only be able to delete data or increase your storage limit. If you reach your RU limit, your cluster will be disabled until the end of the billing cycle unless you increase your RU limit.
 
   {% include cockroachcloud/serverless-usage.md %}
+  
+For multi-region clusters, storage is billed at the same rate, but because there are typically 5 replicas for a piece of data instead of 3, storage effectively costs $0.83 per GiB. Your RU usage will increase for queries that cross regions to account for inter-region bandwidth.
```

Comment:
By default there are three replicas in the primary region and one replica in every non-primary region. So a 6 region cluster would have 3 + 5 = 8 replicas.